uwrt_mars_rover_hw_bridge/CMakeLists.txt

```diff
@@ -0,0 +1,2 @@
+cmake_minimum_required(VERSION 2.8.3)
```

Comment:
you need to add all the other stuff thats normally in a catkin packages cmakelists or else other ros packages wont be able to include it!